packages/app-project/src/screens/SubjectTalkPage/components/SubjectTalkViewer/SubjectTalkViewer.js

```diff
@@ -68,32 +72,38 @@ function SubjectTalkViewer({ subject }) {
         subject={subject}
         src={subjectURL}
       />
-      {locations?.length > 1 ? (
-        <FlipbookControls
-          currentFrame={frame}
-          flipbookSpeed={flipbookSpeed}
-          locations={locations}
-          onFrameChange={setFrame}
-          onPlayPause={onPlayPause}
-          playing={playing}
-          playIterations={1}
-          setFlipbookSpeed={setFlipbookSpeed}
-        />
-      ) : null}
-      {/* <MetaTools /> */}
-      <Box
-        direction='row'
-        gap='small'
-        justify='center'
-        margin='small'
-      >
-        <Favorite />
-        <Bookmark />
-        <ShareOption />
-        <InvertIconButton
-          checked={invert}
-          onClick={onInvert}
-        />
+      <Box>
+        {locations?.length > 1 ? (
+          <FlipbookControls
+            currentFrame={frame}
+            flipbookSpeed={flipbookSpeed}
+            locations={locations}
+            onFrameChange={setFrame}
+            onPlayPause={onPlayPause}
+            playing={playing}
+            playIterations={1}
+            setFlipbookSpeed={setFlipbookSpeed}
+          />
+        ) : null}
+        {/* <MetaTools /> */}
+        <Box
+          direction='row'
+          gap='small'
+          justify='center'
+          margin='small'
+        >
+          <FavoritesIconButton
+            project={project}
+            subject={subject}
+            user={user}
```

Comment:
This can be refactored to just include the necessary properties (i.e. `project.display_name`, `project.id`, `subject.id`, `user.id`, `user.login`), but I'm not sure that's preferable?

Reply:
Or (3rd option*) pass the resource ids and create hooks to get the resources in lib-react-components.

*1st option - current code, pass full resources
2nd option - pass specific resource properties
3rd option - pass resource IDs add hooks to get resources in lib-react-components